dotnet-docs/token-authentication.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Using Tokens for Authentication
+page_title: Authenticate with Tokens against the Report Server for .NET
+description: "Learn how to use JWT Tokens to authenticate against the Telerik Report Server for .NET instance."
+slug: rs-net-token-authentication
+tags: report, server, dotnet, token, authenticate
+published: True
+position: 25
+---
+
+# Configuring the Report Server for .NET for Authentication with JWT Tokens
+
+Starting with [2025 Q4(11.3.25.1112)](https://www.telerik.com/support/whats-new/report-server/release-history/progress-telerik-report-server-2025-q4-11-3-25-1112), Telerik Report Server for .NET enhances security by letting the [Users]({%slug users%}) create Personal JWT Tokens. Each user may have multiple Tokens. The [Guest User]({%slug guest-user%}) may also keep Tokens added by the admin users.
```

Comment:
There is one scenario that we discussed with Miro and Ivan, and we think it should be documented. If a user reinstalls RS.NET without removing the storage or somehow migrates the storage from one RS.NET to another the existing tokens will stop working (because the are sign with different key) but they WILL be visible in the UI as they exist in the storage.

Will see what options we have to remedy this for v2 but for the time being that's how it would behave and it would be nice to have it mentioned in the docs.

Reply:
Added as new restriction